README.md

```diff
@@ -1,10 +1,54 @@
-# containers-ansible
-* using ansible to test container images: rsyslog, support-tools, sadc, rhel-tools, net-snmp, ubi-init/rhel-init, etcd, flannel, ubi base and ubi minimal base images.
-* podman needs to be installed
-* no inventory file included
+# Documentation
 
-# requirements:
-- ansible: 2.6
+- [**Overview**](#overview)
+- [**Requirement**](#requirement)
+- [**Known Issue**](#known-issue)
+- [**Usage**](#usage)
 
-# Usage:
-`ansible-playbook -i $inventory $container.yml -e image_fullname=$img_fn`
+## Overview
+
+Project `containers-ansible` is designed for internal layered image Errata Test.
+
+Currently, project includes rsyslog, support-tools, sadc, rhel-tools, net-snmp, ubi-init/rhel-init, etcd, flannel, ubi base and ubi minimal base images.
+
+## Requirement
+
+- Ansible / ansible-core is necessary in your main (local) node.
+- Podman is necessary in your worker (beaker) node(s).
+  - If worker (beaker) nodes are RHEL-7, book related repos based on different platforms.
+- Add above worker nodes hostname / ip to `inventory` file.
+
+## Known Issue
+
+- Add unsecurity repo task failure
+- Check container size task failure
```

Comment:
If we put this item here, we can describe it a little bit, like why it fails and how we can do when it fails.